driver/pace/driver/run.py

```diff
@@ -9,25 +9,27 @@
 from mpi4py import MPI
 from typing_extensions import Literal
 
-import fv3core._config
 import fv3core.initialization.baroclinic as baroclinic_init
 import pace.util
 from fv3core import DynamicalCore
+from fv3core._config import DynamicalCoreConfig
 from fv3core.initialization.dycore_state import DycoreState
 from fv3core.testing.translate_fvdynamics import init_dycore_state_from_serialized_data
 from fv3gfs.physics import PhysicsState
+from fv3gfs.physics._config import PhysicsConfig
```

Comment:
Can these `Config` classes be imported directly from `fv3core` and from `fv3gfs.physics` at the top level? Some of these other imports could be collapsed down, though `pace.util.grid` (as you know) cannot because of circular dependencies.